xbmc/storage/discs/IDiscDriveHandler.h

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Copyright (C) 2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include <memory>
+#include <string>
+
+/*! \brief Represents the state of a disc (optical) drive
+*/
+enum class DriveState
+{
+  DRIVE_OPEN = 0, ///< The drive is open
```

Comment:
Is there a reason for the explicit initialization?

Reply:
The inline comments clutter things a bit, maybe on the previous line is better?

Reply:
Initialization is a leftover, the documentation I will keep (standard way of documenting enums). I agree it's not the best but it's what we have